### PR TITLE
Fix 2020-03-01 swagger bug.

### PR DESCRIPTION
**Pull request by xingwu1**
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Contribution checklist:
- [x] I have reviewed the [documentation](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/202/Overall-Process-of-AME-Onboarding) for the workflow.
- [x] [Validation tools](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/84/Swagger-Validation-tools) were run on swagger spec(s) and have all been fixed in this PR. [How to fix?](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/ci-fix.md)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/FAQ.md).

### ARM API Review Checklist
- [ ] Service team MUST add the "**WaitForARMFeedback**" label if the management plane API changes fall into one of the below categories. 
  - adding/removing APIs.
  - adding/removing properties.
  - adding/removing API-version. 
  - adding a new service in Azure.

<i>Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.</i>

- [x] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process]( https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/212/Swagger-PR-Review).

### CODE REVIEW

**Comment by xingwu1** [comment]
This is fix original swagger spec error. One of API wasn't set to long run operation.

**Review by bgklein — APPROVED** [review]

**Comment by xingwu1** [comment]
@lirenhe This is kind of breaking change since it redefine the response code.

File: specification/batch/resource-manager/Microsoft.Batch/stable/2020-03-01/BatchManagement.json [hunk 1]
```diff
@@ -1731,11 +1731,22 @@
             "description": "The operation was successful. The response contains the PrivateEndpointConnection.",
             "schema": {
               "$ref": "#/definitions/PrivateEndpointConnection"
-            },
+            }
+          },
+          "204": {
+            "description": "The operation was successful."
+          },
+          "202": {
```
Comment (ravbhatnagar) [inline specification/batch/resource-manager/Microsoft.Batch/stable/2020-03-01/BatchManagement.json:1739]: making a sync call as async is a breaking change and requires a new api-version

**Review by ravbhatnagar — COMMENTED** [review]
@xingwu1 - please take a look. are you change the response code because the operation is changing from sync to async

File: specification/batch/resource-manager/Microsoft.Batch/stable/2020-03-01/BatchManagement.json (hunk 1 above)
Comment (bgklein) [reply specification/batch/resource-manager/Microsoft.Batch/stable/2020-03-01/BatchManagement.json:1739]: To be clear here @ravbhatnagar the underlying API did not change. The previous swagger did not match the FE behavior for the API, which is why these changes are being made.

**Pull request closed without merging**